### PR TITLE
docs(terminology): Updates n/a and -- descriptions for clarity.

### LINKED ISSUE #3754: Update terminology guidance to better differentiate "N/A" and "--"
Follow-up issue for https://github.com/patternfly/patternfly-org/issues/3623

The descriptions for "N/a" and "--" need to be better articulated on the terminology page. This was updated in the mentioned issue, but the new description still leads to confusion, so this can be described more clearly.

### PR DESCRIPTION
**Pull request by edonehoo**
closes #3754

### CODE REVIEW

**Comment by edonehoo** [comment]
@mmenestr wdyt about this tweak? do you think it needs more?

**Comment by mmenestr** [comment]
Hmm I think it's still a little bit confusing but I can't say why, maybe it's the data source thing ...

File: packages/documentation-site/patternfly-docs/content/design-guidelines/content/terminology.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Use abbreviations that users are familiar with, and write out uncommon abbreviat
...
-| **--** | Refers to *data not available*, meaning data could apply to the object, but is currently not populating.<br /><br />This includes scenarios where the object is connected to a data source, but is not able to fetch data. For example, when there are permission restrictions or errors.|
+| **N/A** | Refers to *data not applicable*, meaning data is not relevant or does not currently apply to the object.<br /><br />This includes scenarios where there is no matching data in the object's data source. For example, when nothing matches a filter, when the data source is empty, or when the object is not connected to any data source.|
+| **--** | Refers to *data not available*, meaning data could apply to the object, but is currently not accessible and cannot be obtained at the moment.<br /><br />This includes scenarios where the object is connected to a data source, but is not able to fetch data. For example, when there are permission restrictions or errors.|
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/content/terminology.md:80]:
```suggestion
| **--** | Refers to *data unavailable*, meaning the data exists and fits the criteria, but is not currently available to display.<br /><br />For example, when a data source is connected, but data can't be fetched due to permission restrictions or errors.|
```

File: packages/documentation-site/patternfly-docs/content/design-guidelines/content/terminology.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Use abbreviations that users are familiar with, and write out uncommon abbreviat
...
-| **N/A** | Refers to *data not applicable*, meaning data does not currently apply to the object and will not populate.<br /><br />This includes scenarios where there is no matching data in the object's data source. For example, when nothing matches a filter, when the data source is empty, or when the object is not connected to any data source.|
-| **--** | Refers to *data not available*, meaning data could apply to the object, but is currently not populating.<br /><br />This includes scenarios where the object is connected to a data source, but is not able to fetch data. For example, when there are permission restrictions or errors.|
+| **N/A** | Refers to *data not applicable*, meaning data is not relevant or does not currently apply to the object.<br /><br />This includes scenarios where there is no matching data in the object's data source. For example, when nothing matches a filter, when the data source is empty, or when the object is not connected to any data source.|
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/content/terminology.md:79]:
```suggestion
| **N/A** | Refers to *data not applicable*, meaning there is no relevant, matching, or applicable data.<br /><br />For example, when a filter returns no results, when a data source is empty, or when no data source is connected.|
```

**Comment by edonehoo** [comment]
@mmenestr okay I tried something else.. didn't remove "data source" because I wasn't sure of a better alternative, but if you have ideas for different phrasing there I'm definitely all ears! Katie suggested trying "data unavailable" instead of "data not available" to help separate it more from the other definition more and I think it helps some. How well does it read now?

**Review by mmenestr — APPROVED** [review]

**Pull request merged**